Samples/LibVLCSharp.UWP.Sample/MainPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Windows.UI.Xaml.Controls;
+using LibVLCSharp.Shared;
+
+namespace LibVLCSharp.UWP.Sample
+{
+    /// <summary>
+    /// The main page of the application
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        private LibVLC _libVLC;
+
+        private MediaPlayer _mediaPlayer;
+
+        public MainPage()
+        {
+            InitializeComponent();
+            Loaded += (s, e) =>
+            {
+                _libVLC = new LibVLC(VideoView.GetSwapChainOptions());
```

Comment:
`GetSwapChainOptions` is an implementation detail; it cannot be customized and the users do not need to know about it. I think it should be `internal` and injected automatically in the LibVLC ctor.

Reply:
It can't because the LibVLC instance doesn't know about the video view nor the swapchain...

Reply:
Hence `Document that a swapchain is required in the LibVLC ctor for UWP builds`

Reply:
Documented